Gemfile

```diff
@@ -110,6 +110,8 @@ gem 'newrelic_rpm'
 # Permission helper Gem
 gem 'canaid', git: 'https://github.com/biosistemika/canaid', branch: 'rails_6'
 
+gem 'ruby-handlebars', require: false
```

Comment:
Why do we need yet another templating engine??

Reply:
switched to interpolation